docs/Apple/ws1intelligence.md

```diff
@@ -6,6 +6,8 @@ hide:
   - toc
 ---
 
+[TOC] 
```

Comment:
Should we try using an MkDocs 'table of contents' in this file or take it out?

Reply:
update: we tried it, but the Portal team recommended using a side-bar type of TOC activated in the MkDocs metadata table at the top of the page.